### PR TITLE
Merge annotations and dataset_manager apps

### PR DESCRIPTION
**Pull request by zhiltsov-max**
- **removed** `annotations` module. Its code moved to `dataset_manager` app.
    - `Task-` and `Job-` `Annotation`, moved to `dataset_manager`
    - `Annotation`, `AnnotationIR` moved to `dataset_manager`. `Annotation` is renamed to `TaskData` to reflect its actual status.
    - `data_manager` renamed to `annotation_manager` as no data manipulation took place there
- formats: 
  - dump and dataset formats are made the same, the only difference is image inclusion. All formats are exported as `zip` archives therefore, but other formats are possible.
  - renamed, format names do not include annotation file extension
  - are not stored in the database anymore
  - have terser definition syntax (decorators), can be both classes and functions
  - importers now can accept multiple file formats (e.g. zip or xml)
  - `datumaro` export format for datasets moved to `dump annotations`
  - added `datumaro` export format for datasets, which includes images in the project's cache
- REST API: 
  - removed `dataset/formats`
  - updated output format of `annotation/formats`
  - formats have different serialized representation (removed the `name` field)
  - `dump_annotations` and `export_dataset` have the same logic - **only the recent result is cached** for N hours (10 by default). This was different for annotations, which were cached indefinitely. Downloading still requires the same several API calls, but now for annotations no file name is specified (i.e. old dataset downloading logic is used).
  - updated `GET /task/id/annotations`: parameters are `format` (**mandatory** for export), `filename` (**optional**), `action` (optional)
- General:
  - updated docs in the landing `README` and `user_manual`
  - added auto trimming for trailing whitespaces in the workspace config
- UI:
  - downloaded file name for annotations download in UI changed from task name to task name + update date + format name
- fixed frame step bug (#1294, #1334)
- Datumaro:
  - added `datumaro_project` export format
  - fixed categories for empty projects with no sources
  - added directory removal on error during `extract` command
  - added debug error message on incorrect XPath
  - fixed missing `self` in the cvat format converter (#1494)

Closes https://github.com/opencv/cvat/issues/1334
Closes https://github.com/opencv/cvat/issues/1294
Closes https://github.com/opencv/cvat/issues/1494

### Motivation and context

This PR is part of ongoing efforts on removing deprecated functionality from CVAT. This PR finally removes `annotation` application in favor of more advanced and generic `dataset_manager`.

### How has this been tested?

Unit tests, manual testing.

### Checklist

- [ ] I have raised an issue to propose this change ([required](https://github.com/opencv/cvat/issues))
- [x] My issue has received approval from the maintainers
- [x] I've read the [CONTRIBUTION](https://github.com/opencv/cvat/blob/develop/CONTRIBUTING.md) guide
- [x] I have added description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [x] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
- [x] I have added tests to cover my changes
- [x] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [ ] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2020 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

**Comment by zhiltsov-max** [comment]
- [ ] update format docs
- [ ] update ui
- [ ] update rest api tests
- [ ] update dm tests

**Comment by zhiltsov-max** [comment]
@bsekachev, @nmanovic, @azhavoro, there are 2 REST API calls currently:
- `annotation/formats`, which returns import and export formats for annotations
- `dataset/formats`, which returns export formats for datasets

In this PR it is redone to be:
- `annotation/import_formats`
- `annotation/export_formats`

What do you think on this API? Might it be better or more useful to join and provide only `annotation/formats`?

**Comment by zhiltsov-max** [comment]
Ready for review.

**Comment by nmanovic** [comment]
@azhavoro , could you please review and test the PR today and post your feedback here?

File: cvat-core/src/annotation-format.js [hunk 1]
```diff
@@ -127,108 +105,41 @@
         * @memberof module:API.cvat.classes
         * @hideconstructor
     */
-    class AnnotationFormat {
+    class AnnotationFormats {
```
Comment (bsekachev) [inline cvat-core/src/annotation-format.js:108]: Maybe also rename this file?

File: cvat-ui/src/actions/tasks-actions.ts [hunk 2]
```diff
@@ -169,7 +169,7 @@ ThunkAction<Promise<void>, {}, {}, AnyAction> {
     return async (dispatch: ActionCreator<Dispatch>): Promise<void> => {
         try {
             dispatch(dumpAnnotation(task, dumper));
-            const url = await task.annotations.dump(task.name, dumper);
+            const url = await task.annotations.dump('', dumper);
```
Comment (bsekachev) [inline cvat-ui/src/actions/tasks-actions.ts:172]: Why have you decided left name here as the first argument if it doesn't used anywhere?

File: cvat-ui/src/containers/actions-menu/actions-menu.tsx [hunk 3]
```diff
@@ -125,11 +121,8 @@ function ActionsMenuContainer(props: OwnProps & StateToProps & DispatchToProps):
     } = props;
 
 
-    const loaders = annotationFormats
-        .map((format: any): any[] => format.loaders).flat();
-
-    const dumpers = annotationFormats
-        .map((format: any): any[] => format.dumpers).flat();
+    const loaders = annotationFormats.loaders.flat();
+    const dumpers = annotationFormats.dumpers.flat();
```
Comment (bsekachev) [inline cvat-ui/src/containers/actions-menu/actions-menu.tsx:125]: Probably flat() doesn't needed here anymore. Loaders and dumpers are expected to be flattened.
Just ``const { loaders, dumpers } = annotationFormats;``

File: cvat-ui/src/containers/annotation-page/top-bar/annotation-menu.tsx [hunk 4]
```diff
@@ -113,11 +109,8 @@ function AnnotationMenuContainer(props: Props): JSX.Element {
         installedReID,
     } = props;
 
-    const loaders = annotationFormats
-        .map((format: any): any[] => format.loaders).flat();
-
-    const dumpers = annotationFormats
-        .map((format: any): any[] => format.dumpers).flat();
+    const loaders = annotationFormats.loaders.flat();
+    const dumpers = annotationFormats.dumpers.flat();
```
Comment (bsekachev) [inline cvat-ui/src/containers/annotation-page/top-bar/annotation-menu.tsx:113]: The same as above

**Comment by bsekachev** [comment]
I cannot run server or run ``migrate`` command for example.
I get:
```bash
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/bsekachev/cvat/cvat/apps/auto_annotation/urls.py", line 7, in <module>
    from . import views
  File "/home/bsekachev/cvat/cvat/apps/auto_annotation/views.py", line 20, in <module>
    from . import model_manager
  File "/home/bsekachev/cvat/cvat/apps/auto_annotation/model_manager.py", line 20, in <module>
    from cvat.apps.engine.annotation import put_task_data, patch_task_data
ModuleNotFoundError: No module named 'cvat.apps.engine.annotation'
```
when I am trying:
```bash
    pip install -r cvat/requirements/development.txt
    pip install -r datumaro/requirements.txt
    python manage.py migrate
```

What is wrong?

File: cvat-core/src/annotation-format.js (hunk 1 above)
Comment (zhiltsov-max) [reply cvat-core/src/annotation-format.js:108]: If it makes sense to the code style, I have no preferences here.

**Comment by zhiltsov-max** [comment]
- [ ] Extra applications should be fixed.

File: cvat-core/src/annotation-format.js (hunk 1 above)
Comment (bsekachev) [reply cvat-core/src/annotation-format.js:108]: It's just suggestion. I also do not have preferences here

File: cvat-ui/src/actions/tasks-actions.ts (hunk 2 above)
Comment (zhiltsov-max) [reply cvat-ui/src/actions/tasks-actions.ts]: Argument is made optional

**Comment by nmanovic** [comment]
@zhiltsov-max , need to resolve merge conflicts in user_guide.md. Are all other problems resolved?

File: CHANGELOG.md [hunk 5]
```diff
@@ -6,20 +6,33 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
 - Methods ``import`` and ``export`` to import/export raw annotations for Job and Task in ``cvat-core`` (<https://github.com/opencv/cvat/pull/1406>)
 - Versioning of client packages (``cvat-core``, ``cvat-canvas``, ``cvat-ui``). Initial versions are set to 1.0.0  (<https://github.com/opencv/cvat/pull/1448>)
 
 ### Changed
--
+- Downloaded file name in annotations export became more informative (https://github.com/opencv/cvat/pull/1352)
+- Added auto trimming for trailing whitespaces style enforsement (https://github.com/opencv/cvat/pull/1352)
+- REST API: updated `GET /task/<id>/annotations`: parameters are `format`, `filename` (now optional), `action` (optional) (https://github.com/opencv/cvat/pull/1352)
+- REST API: removed `dataset/formats`, changed format of `annotation/formats` (https://github.com/opencv/cvat/pull/1352)
+- Exported annotations are stored for N hours insted of indefinitely (https://github.com/opencv/cvat/pull/1352)
```
Comment (nmanovic) [inline CHANGELOG.md:19]:
```suggestion
- Exported annotations are stored for N hours instead of indefinitely (https://github.com/opencv/cvat/pull/1352)
```

**Comment by zhiltsov-max** [comment]
@nmanovic, they should be. Both buttons in the old ui and extra apps.

File: cvat-core/src/annotation-format.js (hunk 1 above)
Comment (zhiltsov-max) [reply cvat-core/src/annotation-format.js:108]: Done

**Comment by nmanovic** [comment]
@zhiltsov-max , need to resolve conflicts. @azhavoro , could you please look at the patch?

File: cvat/apps/documentation/user_guide.md [hunk 6]
```diff
@@ -1,3 +1,7 @@
+<!--lint disable maximum-line-length-->
+<!--lint disable list-item-spacing-->
+<!--lint disable list-item-indent-->
+
```
Comment (nmanovic) [inline cvat/apps/documentation/user_guide.md:4]: @zhiltsov-max , please revert these changes and ask @TOsmanov to fix trailing spaces.

File: cvat/apps/engine/static/engine/js/3rdparty/Decoder.worker.js [hunk 7]
```diff
@@ -1,2 +1,2 @@
...
...lice]),0==(b-=1)&&g.length){var a=g.shift();o.decode(new Uint8Array(a.buf,a.offset||0,a.length),a.info,(function(){y&&h!==m&&postMessage(a,[a.buf])}))}return}if(e.data.setSliceCnt)return void P(e.data.sliceCnt)}else e.data&&"Broadway.js - Worker init"===e.data.type&&(l=!0,o=new r(e.data.options),e.data.options.sliceMode?(p=!0,y=!0,h=e.data.options.sliceNum,P(e.data.options.sliceCnt),o.onPictureDecoded=function(e,n,t,r){var a=new Uint8Array(A(e.length));M(e,a,r[0].sliceInfoAr,n),s=r[0].startDecoding,c=r[0].finishDecoding,d=c-s,r[0].timeDecoding=d,r[0].timeCopy=0,postMessage({slice:a.buffer,sliceNum:h,width:n,height:t,infos:r},[a.buffer]),b=v-1,u=e,f=r[0].sliceInfoAr}):e.data.options.reuseMemory?(p=!0,o.onPictureDecoded=function(e,n,t,r){var a=new Uint8Array(A(e.length));a.set(e,0,e.length),postMessage({buf:a.buffer,length:e.length,width:n,height:t,infos:r},[a.buffer])}):o.onPictureDecoded=function(e,n,t,r){e&&(e=new Uint8Array(e));var a=new Uint8Array(e.length);a.set(e,0,e.length),postMessage({buf:a.buffer,length:e.length,width:n,height:t,infos:r},[a.buffer])},postMessage({consoleLog:"broadway worker initialized"}))}),!1)}return r.nowValue=t,r}()})?r.apply(n,a):r)||(e.exports=i)}]);
```
Comment (nmanovic) [inline cvat/apps/engine/static/engine/js/3rdparty/Decoder.worker.js:1]: What was changed here?

File: cvat/apps/engine/static/engine/js/download.worker.js [hunk 8]
```diff
@@ -1,4 +1,4 @@
...
...romise.then((function(e){l&&(l.abort(),c(e),l=null)})),void 0===f&&(f=null),l.send(f)}))}},function(e,t,r){"use strict";var n=r(15);e.exports=function(e,t,r,o,i){var s=new Error(e);return n(s,t,r,o,i)}},function(e,t,r){"use strict";e.exports=function(e){return!(!e||!e.__CANCEL__)}},function(e,t,r){"use strict";function n(e){this.message=e}n.prototype.toString=function(){return"Cancel"+(this.message?": "+this.message:"")},n.prototype.__CANCEL__=!0,e.exports=n},function(e,t,r){const n=r(8);n.defaults.withCredentials=!0,n.defaults.xsrfHeaderName="X-CSRFTOKEN",n.defaults.xsrfCookieName="csrftoken",onmessage=e=>{n.get(e.data.url,e.data.config).then(t=>{postMessage({responseData:t.data,id:e.data.id,isSuccess:!0})}).catch(t=>{postMessage({id:e.data.id,error:t,isSuccess:!1})})}},function(e,t,r){e.exports=r(9)},function(e,t,r){"use strict";var n=r(0),o=r(2),i=r(11),s=r(1);function u(e){var t=new i(e),r=o(i.prototype.request,t);return n.extend(r,i.prototype,t),n.extend(r,t),r}var a=u(s);a.Axios=i,a.create=function(e){return u(n.merge(s,e))},a.Cancel=r(6),a.CancelToken=r(25),a.isCancel=r(5),a.all=function(e){return Promise.all(e)},a.spread=r(26),e.exports=a,e.exports.default=a},function(e,t){
```
Comment (nmanovic) [inline cvat/apps/engine/static/engine/js/download.worker.js:1]: What was changed here?

File: cvat/apps/engine/static/engine/js/unzip_imgs.worker.js [hunk 9]
```diff
@@ -1,9 +1,9 @@
...
...(this.leftOver&&this.leftOver.length){if(i.uint8array){var a=r;(r=new Uint8Array(a.length+this.leftOver.length)).set(this.leftOver,0),r.set(a,this.leftOver.length)}else r=this.leftOver.concat(r);this.leftOver=null}var o=function(t,e){var r;for((e=e||t.length)>t.length&&(e=t.length),r=e-1;r>=0&&128==(192&t[r]);)r--;return r<0||0===r?e:r+s[t[r]]>e?r:e}(r),u=r;o!==r.length&&(i.uint8array?(u=r.subarray(0,o),this.leftOver=r.subarray(o,r.length)):(u=r.slice(0,o),this.leftOver=r.slice(o,r.length))),this.push({data:e.utf8decode(u),meta:t.meta})},h.prototype.flush=function(){this.leftOver&&this.leftOver.length&&(this.push({data:e.utf8decode(this.leftOver),meta:{}}),this.leftOver=null)},e.Utf8DecodeWorker=h,n.inherits(f,o),f.prototype.processChunk=function(t){this.push({data:e.utf8encode(t.data),meta:t.meta})},e.Utf8EncodeWorker=f},function(t,e){"function"==typeof Object.create?t.exports=function(t,e){e&&(t.super_=e,t.prototype=Object.create(e.prototype,{constructor:{value:t,enumerable:!1,writable:!0,configurable:!0}}))}:t.exports=function(t,e){if(e){t.super_=e;var r=function(){};r.prototype=e.prototype,t.prototype=new r,t.prototype.constructor=t}}},function(t,e,r){"use strict";(function(t){
```
Comment (nmanovic) [inline cvat/apps/engine/static/engine/js/unzip_imgs.worker.js:1]: What was changed here?

File: cvat/apps/tf_annotation/views.py [hunk 10]
```diff
@@ -203,7 +203,7 @@ def create_thread(tid, labels_mapping, user):
         result = convert_to_cvat_format(result)
         serializer = LabeledDataSerializer(data = result)
         if serializer.is_valid(raise_exception=True):
-            put_task_data(tid, user, result)
+            put_task_data(tid, result)
```
Comment (nmanovic) [inline cvat/apps/tf_annotation/views.py:206]: @zhiltsov-max , could you please explain why we don't need `user` here anymore? Not all users should be able to run put_task_data function.

Comment (zhiltsov-max) [reply cvat/apps/tf_annotation/views.py:206]: This parameter was never used in any of these functions, or in Task- and JobAnnotations classes.

File: cvat/apps/engine/static/engine/js/unzip_imgs.worker.js (hunk 9 above)
Comment (zhiltsov-max) [reply cvat/apps/engine/static/engine/js/unzip_imgs.worker.js:1]: These files were just rebuilt with `cvat-core` changes.

File: cvat/apps/documentation/user_guide.md (hunk 6 above)
Comment (zhiltsov-max) [reply cvat/apps/documentation/user_guide.md]: @TOsmanov, please, remove trailing spaces from the document.

**Comment by azhavoro** [comment]
I found 2 issues:

1. cannot upload annotation in old UI 
image

2. cannot upload cvat xml file from the downloaded archive, but uploading zip works fine. xml file looks ok
```
Traceback (most recent call last):
  File "/usr/local/lib/python3.5/dist-packages/rq/worker.py", line 812, in perform_job
    rv = job.perform()
  File "/usr/local/lib/python3.5/dist-packages/rq/job.py", line 588, in perform
    self._result = self._execute()
  File "/usr/local/lib/python3.5/dist-packages/rq/job.py", line 594, in _execute
    return self.func(*self.args, **self.kwargs)
  File "/usr/lib/python3.5/contextlib.py", line 30, in inner
    return func(*args, **kwds)
  File "/home/django/cvat/apps/dataset_manager/task.py", line 732, in import_task_annotations
    task.import_annotations(f, importer)
  File "/home/django/cvat/apps/dataset_manager/task.py", line 629, in import_annotations
    importer(src_file, task_data, **options)
  File "/home/django/cvat/apps/dataset_manager/formats/registry.py", line 35, in __call__
    f_or_cls(*args, **kwargs)
  File "/home/django/cvat/apps/dataset_manager/formats/cvat.py", line 557, in _import
    load(src_file, task_data)
  File "/home/django/cvat/apps/dataset_manager/formats/cvat.py", line 416, in load
    ev, _ = next(context)
  File "/usr/lib/python3.5/xml/etree/ElementTree.py", line 1306, in __next__
    self._root = self._parser._close_and_return_root()
  File "/usr/lib/python3.5/xml/etree/ElementTree.py", line 1243, in _close_and_return_root
    root = self._parser.close()
  File "/usr/lib/python3.5/xml/etree/ElementTree.py", line 1652, in close
    self._raiseerror(v)
  File "/usr/lib/python3.5/xml/etree/ElementTree.py", line 1544, in _raiseerror
    raise err
  File "<string>", line None
xml.etree.ElementTree.ParseError: no element found: line 1, column 0
```

**Comment by nmanovic** [comment]
@zhiltsov-max , could you please organise a Skype review of the patch tomorrow?

**Comment by zhiltsov-max** [comment]
@azhavoro, should be fixed now.

File: cvat/apps/engine/static/engine/js/download.worker.js (hunk 8 above)
Comment (zhiltsov-max) [reply cvat/apps/engine/static/engine/js/download.worker.js]: reverted

File: cvat/apps/documentation/user_guide.md (hunk 6 above)
Comment (TOsmanov) [reply cvat/apps/documentation/user_guide.md]: @zhiltsov-max , I remove trailing spaces in this commit https://github.com/opencv/cvat/pull/1508/commits/eaa16551bd8a35f66cd2c7d4dae9a0690b8cb67a

**Comment by zhiltsov-max** [comment]
- [ ] bump cvat-core version

**Comment by azhavoro** [comment]
@nmanovic could you please merge?

**Comment by codewithunknown** [comment]
Hello @nmanovic  @zhiltsov-max 

There is an error when trying to upload annotations with same file name with different extensions 

error:
`Item (image1,'default') is repeated in the source sequence`

when Trying to upload, same file name with different extensions
image


the cvat_server logs
```
2024-03-03 16:47:13 2024-03-03 11:17:13,941 DEBG 'uvicorn-1' stdout output:
2024-03-03 16:47:13 INFO:     172.18.0.1:0 - "POST /api/tasks/4/annotations?org=&use_default_location=true&format=COCO+1.0&filename=anntions.json&conv_mask_to_poly=true HTTP/1.0" 202 Accepted
2024-03-03 16:47:13 
2024-03-03 16:47:14 2024-03-03 11:17:14,004 DEBG 'uvicorn-1' stdout output:
2024-03-03 16:47:14 INFO:     172.18.0.1:0 - "PUT /api/tasks/4/annotations?org=&use_default_location=true&format=COCO+1.0&filename=anntions.json&conv_mask_to_poly=true&rq_id=import:task-4-annotations-by-notimportantupdatesonly HTTP/1.0" 202 Accepted
2024-03-03 16:47:14 
2024-03-03 16:47:17 2024-03-03 11:17:17,085 DEBG 'uvicorn-1' stderr output:
2024-03-03 16:47:17 [2024-03-03 11:17:17,084] ERROR cvat.server.engine: Traceback (most recent call last):
2024-03-03 16:47:17   File "/home/django/cvat/apps/dataset_manager/task.py", line 912, in import_task_annotations
2024-03-03 16:47:17     task.import_annotations(f, importer, conv_mask_to_poly=conv_mask_to_poly)
2024-03-03 16:47:17   File "/home/django/cvat/apps/dataset_manager/task.py", line 796, in import_annotations
2024-03-03 16:47:17     importer(src_file, temp_dir, task_data, **options)
2024-03-03 16:47:17   File "/home/django/cvat/apps/dataset_manager/formats/registry.py", line 36, in __call__
2024-03-03 16:47:17     f_or_cls(*args, **kwargs)
2024-03-03 16:47:17   File "/home/django/cvat/apps/dataset_manager/formats/coco.py", line 37, in _import
2024-03-03 16:47:17     import_dm_annotations(dataset, instance_data)
2024-03-03 16:47:17   File "/home/django/cvat/apps/dataset_manager/bindings.py", line 1929, in import_dm_annotations
2024-03-03 16:47:17     if len(dm_dataset) == 0:
2024-03-03 16:47:17   File "/opt/venv/lib/python3.10/site-packages/datumaro/components/dataset.py", line 820, in __len__
2024-03-03 16:47:17     return len(self._data)
2024-03-03 16:47:17   File "/opt/venv/lib/python3.10/site-packages/datumaro/components/dataset.py", line 544, in __len__
2024-03-03 16:47:17     self.init_cache()
2024-03-03 16:47:17   File "/opt/venv/lib/python3.10/site-packages/datumaro/components/dataset.py", line 338, in init_cache
2024-03-03 16:47:17     for _ in self._iter_init_cache():
2024-03-03 16:47:17   File "/opt/venv/lib/python3.10/site-packages/datumaro/components/dataset.py", line 345, in _iter_init_cache
2024-03-03 16:47:17     for item in self._iter_init_cache_unchecked():
2024-03-03 16:47:17   File "/opt/venv/lib/python3.10/site-packages/datumaro/components/dataset.py", line 460, in _iter_init_cache_unchecked
2024-03-03 16:47:17     raise RepeatedItemError(item_id)
2024-03-03 16:47:17 datumaro.components.errors.RepeatedItemError: Item ('image1', 'default') is repeated in the source sequence.
2024-03-03 16:47:17 
2024-03-03 16:47:17 During handling of the above exception, another exception occurred:
2024-03-03 16:47:17 
2024-03-03 16:47:17 Traceback (most recent call last):
2024-03-03 16:47:17   File "/opt/venv/lib/python3.10/site-packages/rq/worker.py", line 1428, in perform_job
2024-03-03 16:47:17     rv = job.perform()
2024-03-03 16:47:17   File "/opt/venv/lib/python3.10/site-packages/rq/job.py", line 1278, in perform
2024-03-03 16:47:17     self._result = self._execute()
2024-03-03 16:47:17   File "/opt/venv/lib/python3.10/site-packages/rq/job.py", line 1315, in _execute
2024-03-03 16:47:17     result = self.func(*self.args, **self.kwargs)
2024-03-03 16:47:17   File "/home/django/cvat/apps/engine/utils.py", line 332, in import_resource_with_clean_up_after
2024-03-03 16:47:17     result = func(filename, *args, **kwargs)
2024-03-03 16:47:17   File "/usr/lib/python3.10/contextlib.py", line 79, in inner
2024-03-03 16:47:17     return func(*args, **kwds)
2024-03-03 16:47:17   File "/home/django/cvat/apps/dataset_manager/task.py", line 914, in import_task_annotations
2024-03-03 16:47:17     raise CvatImportError(str(ex))
2024-03-03 16:47:17 cvat.apps.dataset_manager.bindings.CvatImportError: Item ('image1', 'default') is repeated in the source sequence.
2024-03-03 16:47:17 
2024-03-03 16:47:17
```

I tried to debug what found that except arises in datumaro and [code link](https://github.com/opencv/cvat/blob/develop/cvat/apps/dataset_manager/task.py#L905-L914)

I am about to fixing this issue, need helpi n  what to do further can you please help me in fixing this 

your help is immensely appreciated

Thank you
